content/docs/gto/why-gto.md

```diff
@@ -0,0 +1,76 @@
+# Why GTO?
+
+**GTO** is a tool for creating an Artifact Registry in your Git repository. One
+of the special cases we would like to highlight is creating a
+[Machine Learning Model Registry](/doc/use-cases/model-registry).
```

Comment:
This whole page is also not in sidebar.json

Reply:
yes

Reply:
Still secretly published in https://mlem.ai/doc/gto/why-gto.